cmd/operator-sdk/bundle/validate.go

```diff
@@ -72,19 +72,32 @@ To build and validate an image:
 
 `,
 		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			jsonOutput := OutPut{}
 			if err = c.validate(args); err != nil {
-				return fmt.Errorf("error validating args: %v", err)
+				msg := "error validating args"
+				if c.isOutputJSON() {
+					jsonOutput.printFatalJSONWith(msg, err)
```

Comment:
we will never return from `printFatalJsonWith` ... so we may want to rename the function to something like `exitFatalJsonWith`?

Reply:
Renamed to fatalJsonWith

Reply:
IMO the fatal make it implicit.